docs/docs/guides/external-library.md

```diff
@@ -1,6 +1,6 @@
 # External Library
 
-This guide walks you through adding an [External Library](../features/libraries#external-libraries).
+This guide walks you through adding an [External Library](../../features/libraries#external-libraries).
```

Comment:
I don't think this is right, why did you change it?

Reply:
It seems wrong to me too, but look at the rendered version of the current version. The link doesn't go where it should.